Assets/Scripts/Controllers/UIController.cs

```diff
@@ -0,0 +1,71 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.UI;
+
+public class UIController : MonoBehaviour {
+	public WorldController wCont;
```

Comment:
I had trouble understanding what wCont was when I first read the code. I suggest changing it to worldController to make the code more readable.